.github/workflows/test.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         strategy:
             matrix:
                 os: [ubuntu-latest, macos-latest]   # eventually add `windows-latest`
-                python-version: [3.8, 3.9, "3.10"]
```

Comment:
i removed 3.8 here to save time and because i think we should drop support soon, maybe on our 0.7 release

Reply:
Do we want to release this now with 3.8 still supported, and deprecate for 0.7? Not sure if 3.8 can be deprecated yet because of wider OS support (need to check in on this)

Reply:
Can we add 3.8 back again?

Reply:
Sure.